sitespeed.io/login.js

```diff
@@ -0,0 +1,24 @@
+const { constructURL } = require("./utils");
+
+module.exports = async function (context, commands) {
+  await commands.cdp.send("Network.setCookie", {
+    domain: context.options.domain,
+    name: "skipsetupintro",
+    value: "true",
+  });
+  await commands.cdp.send("Network.setCookie", {
+    domain: context.options.domain,
+    name: "skipintro",
+    value: "true",
+  });
+  await commands.navigate(constructURL(context, "/dashboard"));
+  await commands.wait.bySelector("input[name='username']", 5000);
```

Comment:
This is nitpicking, but I wonder why you decided not to abstract this to a variable as it's used in 2 places, but did in the other file where a timeout is only used in one place.